CoreScriptsRoot/ServerCoreScripts/ClientChat/ChatSettings.lua

```diff
@@ -0,0 +1,44 @@
+local source = [[
+-- This module is for initial settings that are not intended to be changed during runtime.
+
+local module = {}
+
+module.WindowDraggable = false
+module.WindowResizable = false -- soon (tee emm)
```

Comment:
I would prefer not to ship something that won't work, unless we plan on fixing this very soon